src/workers/commentUpvoteCanceledRep.ts

```diff
@@ -14,7 +19,16 @@ const worker: Worker = {
     try {
       const comment = await con.getRepository(Comment).findOne(data.commentId);
       if (comment.userId !== data.userId) {
-        await increaseReputation(con, logger, comment.userId, -1);
+        const repo = con.getRepository(ReputationEvent);
+        const event = await repo.findOne({
```

Comment:
We don't need `findOne` and then `delete`. We can just send a single delete request